App.js

```diff
@@ -21,7 +21,7 @@ Notifications.setNotificationHandler({
 
 const baseUrl = () => {
   console.log('extra.baseUrl =', Constants.expoConfig.extra.baseUrl);
-  return Constants.expoConfig.extra.baseUrl || "https://staging.timeoverflow.org";
+  return Constants.expoConfig.extra.baseUrl || "https://www.timeoverflow.org";
```

Comment:
I fixed this `baseUrl` in https://github.com/coopdevs/timeoverflow-mobile-app/pull/15. That is how it was supposed to work.